### PR TITLE
Fixes for disktest

### PR DESCRIPTION
**Pull request by narasimhan-v**
* Added capability to get disk as input, and create fs and run tests
* Log file path contains a ';' due to which test might error. Fixed
it
* Updated yaml and Readme
* And some pylint fixes

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

File: io/disk/disktest.py.data/README.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ It writes chunks one by one to the disks (spawning parallel processes per each d
 Available parameters
 --------------------
 
-disks - List of directories of used in test. In case only string is used it's split using ','. When the target is not directory, it's created.
-gigabyte - Disk space that will be used for the test to run.
-chunk_mb - Size of the portion of the disk used to run the test. Cannot be smaller than the total amount of RAM.
+disk            - Disk to be used in test.
+dir             - Directory of used in test. When the target does not exist, it's created.
```
Comment (PraveenPenguin) [inline io/disk/disktest.py.data/README.txt]: please wrap this on 80 char

File: io/disk/disktest.py.data/README.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@ It writes chunks one by one to the disks (spawning parallel processes per each d
...
+gigabyte        - Disk space that will be used for the test to run (Unit - GB).
+chunk_mb        - Size of the portion of the disk used to run the test (Unit MB). Cannot be smaller than the total amount of RAM.
```
Comment (PraveenPenguin) [inline io/disk/disktest.py.data/README.txt]: here as well

Comment (PraveenPenguin) [reply io/disk/disktest.py.data/README.txt]: i figure out whole README.txt  file need to be formatted :)

**Comment by narasimhan-v** [comment]
@PraveenPenguin Thanks

**Pull request merged**